sympy/physics/continuum_mechanics/beam.py

```diff
@@ -2234,3 +2234,64 @@ def deflection(self):
         the three axes.
         """
         return self._deflection
+
+    def helper_function_for_shearplot(self, dir, subs=None):
```

Comment:
It's better to name a helper function with an underscore, like: `_plot_shear_force()`.